pkg/models/operational_solution_subtask.go

```diff
@@ -26,14 +26,14 @@ type OperationalSolutionSubtask struct {
 	baseStruct
 	solutionRelation
 
-	Name   string                           `json:"name"`
-	Status OperationalSolutionSubtaskStatus `json:"status"`
+	Name   string                           `json:"name" db:"name"`
+	Status OperationalSolutionSubtaskStatus `json:"status" db:"status"`
 }
 
 // NewOperationalSolutionSubtask is a constructor to create an instance of OperationalSolutionSubtask
 func NewOperationalSolutionSubtask(
 	createdBy uuid.UUID,
-	ID uuid.UUID,
+	ID uuid.UUID, //TODO: This ID is not being used. Refactor.
```

Comment:
Is this intended to be addressed within this PR or another?

Reply:
This is not meant to be addressed in this feature PR. This is meant to add the note that this can / should be refactored next time someone goes into this file. It isn't actively hurting anything, but it is not needed.